modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -44,6 +62,17 @@ const ACTION_TYPES = [
 	},
 ];
 
```

Comment:
Changed due to https://github.com/liferay-frontend/liferay-portal/pull/3765#discussion_r1369889907